src/coreclr/gc/unix/gcenv.unix.cpp

```diff
@@ -137,32 +137,13 @@ typedef cpuset_t cpu_set_t;
 
 #if HAVE_NUMA_H
 
-#include <numa.h>
+#include <../../pal/src/numa/numashim.h>
```

Comment:
Including Win32 PAL headers from here is going to be a mess. I do not think we want to make the GC PAL depend on Win32 PAL.

Reply:
hmm, it appears to be working, but agree that this could be problematic. Will undo.